### PR TITLE
Added Advice Generator App - Frontend Mentor Challenge Shiba-Tatsuya

### PR DESCRIPTION
**Pull request by Jatin-Rane-007**
Hi there,

I've added my solution to the Frontend Mentor "Advice generator app" challenge to this repository as part of Hacktoberfest. This solution helps users generate a piece of advice by clicking the dice icon and is built to be responsive on various device screen sizes.

## Details
- Added HTML, CSS, and JavaScript files to implement the solution.
- Ensured that the layout is responsive and adapts to different screen sizes.
- Included hover states for all interactive elements on the page.
- Provided screenshots of the solution in both mobile and desktop views.

## Solution URLs
- [GitHub Repository](https://github.com/Shiba-Tatsuya/advice_generator)
- [Live Demo](https://shiba-tatsuya.github.io/advice_generator/)

desktop
mobile

I hope this solution meets the requirements of the challenge and aligns with the coding standards of this repository. Please feel free to review and provide any feedback or suggestions.

Thank you!

Best regards,
[RANE JATIN](https://github.com/Shiba-Tatsuya)

### CODE REVIEW

**Comment by PhantomOz** [comment]
Please add your name to the name section of Contributors.md and Title of the project you worked on

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -6,5 +6,6 @@ This file lists the contributors who have completed the frontend challenges from
 | Name | GitHub Profile | Challenge |
 | --- | --- | --- |
 | John Doe | [johndoe] | Profile card component |
+| John Doe | [Shiba-Tatsuya](https://github.com/Shiba-Tatsuya) | Profile card component |
```
Comment (PhantomOz) [inline CONTRIBUTORS.md]: Replace John Doe with your name
Replace Profile card component with your name also

**Review by PhantomOz — CHANGES_REQUESTED** [review]

File: CONTRIBUTORS.md (hunk 1 above)
Comment (Jatin-Rane-007) [reply CONTRIBUTORS.md]: sir I changed contributors.md according to you please check

**Comment by Jatin-Rane-007** [comment]
sir I changed contributors.md according to you please check

**Review by PhantomOz — APPROVED** [review]

**Pull request merged**